pkg/domain/vulnerability.go

```diff
@@ -156,19 +156,20 @@ type WebVulnerability struct {
 	ServiceID       int32                      `json:"service_id,omitempty"`
 	SolutionAdvice  string                     `json:"solution_advice"`
 	Reference       string                     `json:"reference"`
-	OtherInfo       string                     `json:"other_info"`
-	WascID          *string                    `json:"wasc_id,omitempty"`
 	CreatedAt       *time.Time                 `json:"created_at"`
 	UpdatedAt       *time.Time                 `json:"updated_at"`
 	Instances       []WebVulnerabilityInstance `json:"instances"`
+	CweID           string                     `json:"cwe_id,omitempty"`
+	WascID          string                     `json:"wasc_id,omitempty"`
 }
 
 type WebVulnerabilityInstance struct {
-	URI      string `json:"uri"`
-	Method   string `json:"method"`
-	Param    string `json:"param"`
-	Attack   string `json:"attack"`
-	Evidence string `json:"evidence"`
+	URI       string `json:"uri"`
+	Method    string `json:"method"`
+	Param     string `json:"param"`
+	Attack    string `json:"attack"`
+	Evidence  string `json:"evidence"`
+	OtherInfo string `json:"other_info"`
 }
```

Comment:
`[Issue]` Since this is embedded into our DTO, we seem to be missing two fields: "id" and "source_vulnerability_id"

<img width="812" height="345" alt="Image" src="https://github.com/user-attachments/assets/ce768e2d-aa6e-4b2e-b8de-b7a45ca6addb" />


```suggestion
        ID        string `json:"string"`
        SourceVulnerabilityID string `json:"source_vulnerability_id"`
	URI       string `json:"uri"`
	Method    string `json:"method"`
	Param     string `json:"param"`
	Attack    string `json:"attack"`
	Evidence  string `json:"evidence"`
	OtherInfo string `json:"other_info"`
```

Reply:
Yes, I forgot to add this fields, I would make double check for the dots.